Formula/soci.rb

```diff
@@ -16,7 +16,11 @@ class Soci < Formula
   depends_on "sqlite"
 
   def install
+    # NOTE: SOCI_TESTS disabled temporarily on macOS M1 machines as
+    # soci 4.0.1 relies on an older version of Catch which fails to
+    # detect macOS machines running on arm64 architecture
```

Comment:
We don't need to build tests so we should probably just keep it disabled permanently. I don't see any reason to re-enable it again.

Reply:
Good point; thanks! I've dropped the comment.